### ISSUE TITLE
Auto ML Model Builder Object detection - bad or no results

### ISSUE THREAD

**Issue by hannespreishuber** [dotnet/machinelearning#6990]
Windows 11 Surface Pro 8 
 - ML.NET Version 3.01
 -NET 8

I have a selfmade training set of free hand drawn simple objects - black on white - circle, rect, triangle. Training with AutoML image classification works fast and detection results are very good.

I did a dataset vott image tagging. Created a single json file and ~150 images. Training with AutoML and Object Detection. Takes ~25minutes and results are unusable.
MAP50_95: 0,1258
Modell: ObjectDetectionMulti

Expected results is a better/good detection rate. I have no idea whats going on with my dataset. And how optimize it.
Side effect: Error Message from Visual Studio "no bbox detected" have to restart VS to rerun..

attached Json and sample image
[916a019c0896a9c18f5b7b2540b8014b-asset.json](https://github.com/dotnet/machinelearning/files/14191867/916a019c0896a9c18f5b7b2540b8014b-asset.json)
![dreieck0](https://github.com/dotnet/machinelearning/assets/3587156/af685623-d45a-431c-a074-e35f08055382)

**Comment by michaelgsharp** [comment]
@LittleLittleCloud can you take a look at this since its run through automl?

**Comment by LittleLittleCloud** [comment]
@hannespreishuber model builder doesn't actually run AutoML optimizing on deep learning scenarios like object detection, so maybe you need to adjust hyper-parameters manually. For example, adding more epoches.

In the meantime, could you share dataset with us as well? It's possible that the poor metric is also dataset-related.

**Comment by hannespreishuber** [comment]
I am quite sure there is something I dont know in the dataset- My point is that Auto- doenst guide me in this 
[vott-json-export.zip](https://github.com/dotnet/machinelearning/files/14205339/vott-json-export.zip)
